plugins/automation/lib/discourse_automation/scripts.rb

```diff
@@ -14,6 +14,7 @@ module Scripts
     GROUP_CATEGORY_NOTIFICATION_DEFAULT = "group_category_notification_default"
     PIN_TOPIC = "pin_topic"
     POST = "post"
+    REMOVE_DELETED_POST_UPLOADS = "remove_deleted_post_uploads"
```

Comment:
If we rename above, this likely should be renamed too, to `remove_upload_markup_from_deleted_posts`.